src/genericComponents/Input.tsx

```diff
@@ -132,40 +158,22 @@ const InputField = (props: any) => {
 
   return (
     <>
-      <CAGInput>
+      <CAGFormGroup controlId={name}>
         <CAGLabel>{label}</CAGLabel>
-        <Form.Control
+        <CAGFormControl
           aria-label={label}
           name={name}
           onChange={onChange}
           placeholder={placeholder}
           type={fieldType}
           {...rest}
+          style={{ height: 40, maxWidth: 256 }}
         />
-      </CAGInput>
-      {hasError ? (
-        <CAGError>{errorMessage || ErrorMessage.Default}</CAGError>
-      ) : (
-        <></>
-      )}
+      </CAGFormGroup>
+      {hasError && <CAGError>{errorMessage || ErrorMessage.Default}</CAGError>}
```

Comment:
You'll want to run through the individual sign up to make sure this doesn't have any weird side effects there, but looks like a fine update